modules/end-user-guide/partials/proc_che-including-kubernetes-applications-in-a-workspace-devfile-definition.adoc

```diff
@@ -34,47 +34,13 @@ metadata:
 +
 For example, to embed the NodeJS-Mongo application in the `minimal-workspace`:
 +
-[source,yaml]
-----
-apiVersion: 1.0.0
-metadata:
- name: minimal-workspace
-components:
- - type: kubernetes
-   reference: https://raw.githubusercontent.com/.../mongo-db.yaml
- - alias: nodejs-app
-   type: kubernetes
-   reference: https://raw.githubusercontent.com/.../nodejs-app.yaml
-   entrypoints:
-     - command: ['sleep']  <1>
-       args: ['infinity']
-----
+include::example$snip_including-kubernetes-applications-in-a-workspace-devfile-definition-nodejs-minimal.adoc[]
```

Comment:
If the examples are not context specific, there is no need to put them in a separate file.